src/ui/common/src/components/pages/metric/id/index.tsx

```diff
@@ -0,0 +1,462 @@
+import { faCircleCheck } from '@fortawesome/free-solid-svg-icons';
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { CircularProgress, Link, List, ListItem } from '@mui/material';
+import Accordion from '@mui/material/Accordion';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import React, { useEffect, useState } from 'react';
+import Plot from 'react-plotly.js';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link as RouterLink, useNavigate, useParams } from 'react-router-dom';
+
+import StickyHeaderTable from '../../../../components/tables/StickyHeaderTable';
+import { boolArtifactNodeIcon } from '../../../../components/workflows/nodes/BoolArtifactNode';
+import { checkOperatorNodeIcon } from '../../../../components/workflows/nodes/CheckOperatorNode';
+import { databaseNodeIcon } from '../../../../components/workflows/nodes/DatabaseNode';
+import { dictArtifactNodeIcon } from '../../../../components/workflows/nodes/DictArtifactNode';
+import { functionOperatorNodeIcon } from '../../../../components/workflows/nodes/FunctionOperatorNode';
+import { genericArtifactNodeIcon } from '../../../../components/workflows/nodes/GenericArtifactNode';
+import { imageArtifactNodeIcon } from '../../../../components/workflows/nodes/ImageArtifactNode';
+import { jsonArtifactNodeIcon } from '../../../../components/workflows/nodes/JsonArtifactNode';
+import { metricOperatorNodeIcon } from '../../../../components/workflows/nodes/MetricOperatorNode';
+import { numericArtifactNodeIcon } from '../../../../components/workflows/nodes/NumericArtifactNode';
+import { stringArtifactNodeIcon } from '../../../../components/workflows/nodes/StringArtifactNode';
+import { tableArtifactNodeIcon } from '../../../../components/workflows/nodes/TableArtifactNode';
+import { NodeType } from '../../../../reducers/nodeSelection';
+import {
+  handleGetArtifactResults,
+  handleGetOperatorResults,
+  handleGetWorkflow,
+} from '../../../../reducers/workflow';
+import { AppDispatch, RootState } from '../../../../stores/store';
+import { ArtifactType } from '../../../../utils/artifacts';
+import UserProfile from '../../../../utils/auth';
+import { Data } from '../../../../utils/data';
+import { getPathPrefix } from '../../../../utils/getPathPrefix';
+import { LoadingStatusEnum } from '../../../../utils/shared';
+import DefaultLayout from '../../../layouts/default';
+import { LayoutProps } from '../../types';
+
+type MetricDetailsHeaderProps = {
+  artifactName: string;
+  createdAt?: string;
+};
+
+const MetricDetailsHeader: React.FC<MetricDetailsHeaderProps> = ({
```

Comment:
Since this is shared with the Artifact details page, let's create a shared component "DetailsPageHeader" and use that.